kubernetes/config/storage.yaml

```diff
@@ -30,7 +30,7 @@ data:
         listenAddresses: [':8981'],
         authenticationPolicy: { allow: {} },
       }],
-      allowAcUpdatesForInstances: ['bb-event-service', 'remote-execution'],
+      allowAcUpdatesForInstanceNamePrefixes: ['bb-event-service', 'remote-execution'],
```

Comment:
If you want, you can change this to use the empty string as well:
```
allowAcUpdatesForInstanceNamePrefixes: [''],
```

Reply:
done